components/brave_wallet/browser/fil_transaction.cc

```diff
@@ -78,7 +79,11 @@ std::optional<FilTransaction> FilTransaction::FromTxData(
 
   auto address = FilAddress::FromAddress(tx_data->to);
   if (address.IsEmpty()) {
-    address = FilAddress::FromFEVMAddress(is_mainnet, tx_data->to);
+    auto eth_address = EthAddress::FromHex(tx_data->to);
+    if (!eth_address.IsValid()) {
```

Comment:
Are there plans to get rid of IsValid methods and use expected\optionl instead?

Reply:
That's something I wanted to do but would be too much for this PR